README.md

```diff
@@ -19,31 +23,37 @@ target 'NinchatClient' do
     pod 'NinchatSDK', :git => 'https://github.com/somia/ninchat-sdk-ios', :branch => 'swift'
 end
 ```
+**Don't forget to specify *swift* branch for `NinchatSDK`. Otherwise, the project fails to build.**
```

Comment:
This is no longer the case?

Reply:
It is. Swift SDK is using a specific branch of `NinchatSDK` which provides support for swift integration.

Reply:
Ah sorry. I didn't pay attention.